cli/src/commands/git/clone.rs

```diff
@@ -69,9 +70,11 @@ pub struct GitCloneArgs {
     /// Create a shallow clone of the given depth
     #[arg(long)]
     depth: Option<NonZeroU32>,
-    /// Configure when to fetch tags
-    #[arg(long, value_enum, default_value_t = FetchTagsMode::Included)]
-    fetch_tags: FetchTagsMode,
+    /// Configure when to fetch tags. Unless otherwise specified, the initial
+    /// clone will fetch all tags, while all subsequent fetches will only fetch
+    /// included tags.
```

Comment:
nit: keep the short help message short (and remove period from the first line.)

```
    /// Configure when to fetch tags
    ///
    /// Unless otherwise specified, the initial clone will fetch all tags, while
    /// all subsequent fetches will only fetch included tags.
```

Reply:
Done!